doc/dftb+/manual/dftbp.tex

```diff
@@ -4426,38 +4426,46 @@ \subsubsection{DFTB3-D3H5}
 }
 \end{verbatim}
 
-\subsection{RangeSeparated}
-\label{sec:dftbp.RangeSep}
+\subsection{Hybrid}
+\label{sec:dftbp.Hybrid}
 
-The \kw{RangeSeparated} keyword specifies the use of a range separated hybrid
-functional. Currently, only the long-range corrected hybrid functional (\is{LC})
-\cite{niehaus-PSSB-249-237,lutsker-JCP-143-184107} is implemented. There, the
-electrostatic interaction is split up into long and short ranged components
-according to
+The \kw{Hybrid} keyword specifies the use of a (range-separated) hybrid functional.
+For the non-periodic formalism we refer to
+Refs.~\cite{niehaus-PSSB-249-237,lutsker-JCP-143-184107}, whereas the formalism for
+periodic boundary conditions is developed in Ref.~\cite{vdH-JCP-???-??????}.
+The electron-electron interaction is partitioned into short- and long-range contributions
 \begin{equation*}
-\frac{1}{r}=\frac{1-e^{-\omega r}}{r}+\frac{e^{-\omega r}}{r},
+\frac{1}{r} = \underbrace{\frac{1 - (\alpha + \beta)
++ \beta \mathrm{e}^{-\omega r}}{r}}_{\text{DFT}} + \underbrace{\frac{\alpha
++ \beta (1 - \mathrm{e}^{-\omega r})}{r}}_{\text{HF}},
 \end{equation*}
-with the range-separation parameter $\omega$, which is set in the Slater-Koster
-files. The option should only be used with corresponding parameter sets created
-for use with long-range corrections.
-
-The \kw{RangeSeparated} keyword expects either \is{None} (default -- no use of
-range-separated hybrid functional) or the \is{LC}\cb\ block as value. The latter
-has the following option:
+where the short-range part is handled by a modified pure density functional, while
+the exact Fock exchange enforces the correct asymptotic decay in the long-range.
+The parameters $\alpha, \beta$ and $\omega$ determine the fraction of global and
+long-range Fock exchange, as well as the value of the smooth range-separation function,
+that is of Yukawa type.
+$\alpha, \beta$ and $\omega$ are set in the Slater-Koster files. If the provided
+parametrization was not created for the respective hybrid xc-functional,
+\dftbp{} will issue an error message and halt.
+
+The \kw{Hybrid} keyword expects either \is{None} (default -- no use of hybrid functional),
```

Comment:
Definitely need an explantion of the different functionals (possibly in the table suggested elsewhere for the manual).

Reply:
I have extended the corresponding manual entry and tried to guide the user, depending on the boundary conditions and further requirements (MPI, colinear spin, restart capability, etc.).